federatedscope/core/monitors/monitor.py

```diff
@@ -376,6 +376,24 @@ def format_eval_res(self,
                                 all_res.size // 10]
                             new_results[f"{key}_top_decile"] = all_res[
                                 all_res.size * 9 // 10]
+                            # log more fairness metrics
+                            
+                            # min and max
+                            new_results[f"{key}_min"] = np.mean(all_res[0])
```

Comment:
why np.mean()?   all_res[0] or all_res[-1] should be only one element, i.e., the performance of one client